### ISSUE TITLE
Support for jax.scipy?

### ISSUE THREAD

**Issue by jonbarron** [keras-team/keras#19549]
Hi all, is there a plan to add support for some of scipy? When writing JAX code, there are many functions like sigmoid and softmax that one must either call `jax.nn.*` for, or `jax.scipy.special.*` for. The suggested approach for Keras 3 seems to be to call `keras.ops.sigmoid`, but this results in code that is no longer backwards compatible with a no-Keras JAX implementation. What I'd like to be able to do is write code like:

```
import keras.ops as jnp
import keras.scipy as jsp

jsp.special.sigmoid(jnp.zeros(10))
```

This code will still work if I switch back from keras to jax.numpy and jax.scipy as a backend, but this isn't the case if I use Keras 3 (unless I gain a Keras 3 dependency, which seems like a lot for just some sigmoids and logits).

**Comment by fchollet** [comment]
Thanks for the suggestion. However, that's not in scope for Keras. Keras does not aim at providing a drop-in replacement for NumPy that you could run with an existing codebase. The reason we adopt the NumPy API standard is to minimize user surprise, not to be code-compatible with NumPy.

For your use case, note `sigmoid` is exported at `keras.ops.nn.sigmoid`, which matches `jax.nn.sigmoid` (albeit not intentionally).
